cli/command/stack/deploy.go

```diff
@@ -35,6 +36,9 @@ func newDeployCommand(dockerCli *command.DockerCli) *cobra.Command {
 		Args:    cli.ExactArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
 			opts.namespace = args[0]
+			if strings.HasSuffix(opts.namespace, ".dab") {
+				opts.namespace = opts.namespace[:len(opts.namespace)-4]
```

Comment:
would using [`TrimSuffix`](https://golang.org/pkg/strings/#TrimSuffix) be more readable to use here?